libpam_internal/pam_debug.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * This provides the necessary functions to do debugging in PAM.
+ * Cristian Gafton <gafton@redhat.com>
+ */
+
+#include "config.h"
+
+#ifdef PAM_DEBUG
+
+#include <stdio.h>
+#include <sys/types.h>
+#include <stdarg.h>
+#include <errno.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "security/_pam_types.h"
+
+/*
+ * This is for debugging purposes ONLY. DO NOT use on live systems !!!
+ * You have been warned :-) - CG
+ *
+ * to get automated debugging to the log file, it must be created manually.
+ * _PAM_LOGFILE must exist and be writable to the programs you debug.
+ */
+
+#ifndef _PAM_LOGFILE
+#define _PAM_LOGFILE "/var/run/pam-debug.log"
+#endif
+
+#ifdef UNUSED
+UNUSED
+#endif
```

Comment:
Let's use `UNUSED` unconditionally in all files besides installed header files.

Reply:
And adjusted as well

Reply:
Oh, didn't notice that this comment was directed at pam_debug.c. Now it's adjusted.